src/web/server/render.ts

```diff
@@ -52,8 +58,8 @@ export const render = (
 			.validateAsCAPIType()
 			.addDividers()
 			.enhanceBlockquotes()
+			.addDots()
```

Comment:
I think `enhanceDots` is better in that it follows the established lexicon where we take an existing thing and enhance it